test/test_monitor.js

```diff
@@ -252,6 +252,35 @@ describe('Collect metrics middleware', () => {
     })
   });
 
+  it('should set isError to false when not contains error message - collectDependencyTime', async() => {
+    const start = process.hrtime()
+    Monitor.collectDependencyTime("dependencyNameTest", "fooType", 304, "GET", "/test", "", start)
```

Comment:
@KarineValenca, I think that even better would be for us to `chai.request` some endpoint of our app_test that collects dependency time in there. Collecting dependency time directly looks far from reality.

Reply:
@abilioesteves  Since the focus of this test case is to test the collectDependencyTime function, I thought it would be better to test this function isolated.

Reply:
Ok. We review our test strategy in another issue. Thank you!